api/src/main/resources/org/openmrs/api/db/hibernate/OrderSet.hbm.xml

```diff
@@ -35,6 +35,8 @@
                 <param name="useNamed">true</param>
             </type>
         </property>
+        <property name="category" type="java.lang.String"
```

Comment:
Can you show me where you defined category to be of type String?

Reply:
I don't think I defined it. I would like to know if that then means I should have it declared in the orderSet.java file, as well as change the type I declared in the liquibase changeset?

Reply:
Does the ticket say that the category field is a String?

Reply:
It's not really mentioned what types to use but according to description, I see that char would be the type to use in this case

Reply:
Have you googled to read about hibernate mapping files?

Reply:
Yes I have, that's how I came up with code for the hibernate mapping, maybe not as specifically on types and related but coming up with a mapping file to add the category field

Reply:
From the documentation that you read, where does not type come from?

Reply:
Sorry for late response but the doc I read said the types come from the class

Reply:
Which is correct. Does your class say that the type is String?

Reply:
I have also been trying to put the type for the field in the mapping file but I got some build fails. I used int(11) as well since in the Database Concept is of type int(11)

Reply:
Have you looked at other existing openmrs hibernate mapping files for classes that have the same type as yours?

Reply:
I haven't noticed any, I'll have to re-look through